pkg/vulnsrc/vulnerability/types.go

```diff
@@ -43,10 +43,10 @@ func NewSeverity(severity string) (Severity, error) {
 	return SeverityUnknown, fmt.Errorf("unknown severity: %s", severity)
 }
 
-func CompareSeverityString(sev1, sev2 string) bool {
+func CompareSeverityString(sev1, sev2 string) int {
 	s1, _ := NewSeverity(sev1)
 	s2, _ := NewSeverity(sev2)
-	return s1 < s2
+	return int(s2) - int(s1)
```

Comment:
Should an `Unknown` severity be ranked higher than a known `Low` one, if the `Low` Vulnerability has a `PkgName` set? I see this is the current behaviour but I'm just probing to see if it is what we want. You can see a test that demonstrates this here: https://github.com/aquasecurity/trivy/pull/201/commits/895b7312075eb8358a06ac5b5cea50dffbc1e49f